modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,6 @@
 * xref:home:index.adoc[Docs Home]
 * xref:get-started:index.adoc[Get Started]
-** xref:get-started:whats-new-233.adoc[]
-** xref:get-started:install-beta.adoc[]
```

Comment:
If we want to keep this content around, we should move it to a partial and delete the page. Partials don't get published to the site.

Reply:
@JakeSCahill I removed those 2 files from `nav.adoc`, but I didn't delete them. We can pull `install-beta.adoc` back in next Beta release, and use `whats-new-233.adoc` for the next What's New in xx Beta release. (That file should be renamed from 233 to beta.) Does that sound right?

Reply:
We should delete the files, otherwise they'll be published. If we want to keep them around without publishing, we can move them to the `partials` directory.

Reply:
I moved `install-beta.adoc` to `get-started/partials` and added a redirect for it in the Deploy index page